### ISSUE TITLE
[DEV] Unable print useful msg for 'curl is not installed'  when run bmcdiscover on Ubuntu

### ISSUE THREAD

**Issue by xuweibj** [xcat2/xcat-core#3100]
OS: Unbuntu 16.04.1
Arch: PPC64LE

When run ``bmcdiscover -s nmap --range x.x.x.x -u root -p 0penBmc``, return nothing.

More detail:
```
# XCATBYPASS=1 bmcdiscover -s nmap --range x.x.x.x -u root -p 0penBmc
sh: 1: curl: not found
```

**Comment by xuweibj** [comment]
The issue was fixed in pull request #3122

**Comment by xuweibj** [comment]
Verified on Ubuntu 16.04
```
# bmcdiscover -s nmap --range 10.200.0.1 -u root -p 0penBmc
10.200.0.1,SIMULATER,Y130UF72701M,root,0penBmc,mp,bmc
```
